### PR TITLE
Update TA guide

### PR DESCRIPTION
**Pull request by mxie**
Sounds like it was copied and pasted from a very old email.

The pep talk has been tweaked, and the links have been updated. Also renamed the file.

### CODE REVIEW

**Comment by mxie** [comment]
@eudaimonious - I know that you send some stuff out as well that we could also include here. Not sure if you might want to put those things in the repo yourself or just include it here. Let me know!

**Comment by mxie** [comment]
Any thoughts on this @eudaimonious? @gabebw maybe?

File: ta-guide.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+This is a guide for our TA's on how to prepare for the workshop.
+
+First off though, thanks for volunteering and making this happen. Over the years
+of running these workshops, we've consistently gotten feedback on how amazing
+the TA's were. We're very excited to have you be a part of this experience.
+
+For machine set-up on Friday, please review the installfest guide so you'll be
+prepared to help our students: <http://docs.railsbridgeboston.org/installfest>
```
Comment (gabebw) [inline ta-guide.md]: `<URL>` in markdown gets rendered as just the URL (at least on GitHub). How about removing the `<`, `>`?

Will we be sending the URL for this document to all the TAs?

Comment (mxie) [inline ta-guide.md]: Sure that’s fine.

It’s a bit late to send to TA’s right now maybe…but definitely for the future we can reference this.

Comment (mxie) [inline ta-guide.md]: Fixed in  1cff986

**Comment by gabebw** [comment]
this is rad :+1:

**Pull request merged**